app/test_main.py

```diff
@@ -1,3 +1,41 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_raise_error_if_age_is_not_number() -> None:
+    with pytest.raises(TypeError):
+        get_human_age(10, "10")
+
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_age",
+    [
+        (-1, -1, [0, 0]),
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
```

Comment:
Looks good, but I advise you to make some changes to these data, in the first you can use both 27, 28 values,  in the second also you can use both values which will give 3 in the result